src/daemon/http/server.rs

```diff
@@ -593,7 +592,7 @@ pub async fn rfc8181(req: Request) -> RoutingResult {
         };
 
         let read = state.read().await;
-        match read.rfc8181(publisher, bytes, &actor) {
+        match read.rfc8181(publisher, bytes) {
```

Comment:
Why is actor no longer passed here?

Reply:
Ah, it was event based before and so an actor was propagated to log as the initiator of the event.